### ISSUE TITLE
Spyder launch error

### ISSUE THREAD

**Issue by Saadi4469** [spyder-ide/spyder#14055]
I am getting the following error while launch Spyder 3.3.6 from Anaconda:
```
Traceback (most recent call last):
File "D:\ProgramData\Anaconda3\envs\py2\lib\site-packages\spyder\app\mainwindow.py", line 3285, in main
mainwindow = run_spyder(app, options, args)
File "D:\ProgramData\Anaconda3\envs\py2\lib\site-packages\spyder\app\mainwindow.py", line 3138, in run_spyder
main = MainWindow(options)
File "D:\ProgramData\Anaconda3\envs\py2\lib\site-packages\spyder\app\mainwindow.py", line 377, in __init__
from spyder.plugins.shortcuts import ShortcutsConfigPage
File "D:\ProgramData\Anaconda3\envs\py2\lib\site-packages\spyder\plugins\shortcuts.py", line 835, in 
class ShortcutsConfigPage(GeneralConfigPage):
File "D:\ProgramData\Anaconda3\envs\py2\lib\site-packages\spyder\plugins\shortcuts.py", line 839, in ShortcutsConfigPage
ICON = ima.icon('keyboard')
File "D:\ProgramData\Anaconda3\envs\py2\lib\site-packages\spyder\utils\icon_manager.py", line 266, in icon
directory=_resource['directory'])
File "D:\ProgramData\Anaconda3\envs\py2\lib\site-packages\qtawesome\__init__.py", line 178, in load_font
return _instance().load_font(prefix, ttf_filename, charmap_filename, directory)
File "D:\ProgramData\Anaconda3\envs\py2\lib\site-packages\qtawesome\__init__.py", line 54, in _instance
'materialdesignicons-webfont-charmap.json')
File "D:\ProgramData\Anaconda3\envs\py2\lib\site-packages\qtawesome\iconic_font.py", line 206, in __init__
super().__init__()
TypeError: super() takes at least 1 argument (0 given)
```
I did try to reset Spyder but it did not work and I tried removing Spyder by conda remove Spyder (and then reinstalling via conda), but both of these steps don't solve the issue.

**Comment by ccordoba12** [comment]
Hey @Saadi4469, thanks for reporting, To fix this error, please install QtAwesome **0.7.3** or start using Python 3 (Python 2 reached end of life at the beginning of this year, sorry).

**Comment by Saadi4469** [comment]
Updating Spyder fixed the issue!

Kind Regards,
Saad Baloch
